### ISSUE TITLE
Align standards and formats across all charts

### ISSUE THREAD

**Issue by flxw** [sigstore/helm-charts#536]
Each of the Sigstore Helm charts has its own lineage and may not have the same standards and format holistically as others. Seek to define and provide alignment amongst all Sigstore charts.

**Acceptance criteria:**
* All charts offer the same options and parameters so that it's less likely to make mistakes during the configuration

**Comment by bobcallaway** [comment]
love this idea - we also probably need to make sure that these helm-charts stay general purpose, and things we've found we need added for the public deployment are configurable VS hard coded.

**Comment by hectorj2f** [comment]
@sabre1041 has been working on a similar initiative. Right @sabre1041 ?

**Comment by sabre1041** [comment]
@hectorj2f yup. been throwing some ideas with @flxw in this [doc](https://docs.google.com/document/d/1dAiy69DYb2me-kmGbGQZNPsPLDpG__1WIuJAfqRNrSg) describing the v2 iteration of charts for the project
